src/ts/components/subtitleoverlay.ts

```diff
@@ -86,10 +101,89 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
       }
     });
 
+    this.configureCea608Captions(player, uimanager);
     // Init
     subtitleClearHandler();
   }
 
+  configureCea608Captions(player: bitmovin.PlayerAPI, uimanager: UIInstanceManager): void {
+    let ratio = 1;
+    let generateCea608Ratio = () => {
+      let dummyText = 'aaaaaaaaaa';
+      let label = new SubtitleLabel({
+        // One letter label used to calculate the height wifth ratio of the font
+        // Works because we are using a monospace font for cea 608
+        // Using a longer string increases precision due to width being an integer
+        text: dummyText,
+      });
+      let domElement = label.getDomElement();
+      domElement.addClass(this.prefixCss(SubtitleOverlay.CLASS_CEA_608));
```

Comment:
The class must be added to the overlay, not to the label. Please take a look at the CSS to understand why.